src/cmd/services/m3coordinator/ingest/m3msg/ingest.go

```diff
@@ -139,19 +144,21 @@ func (i *Ingester) Ingest(
 }
 
 type ingestOp struct {
-	s         storage.Appender
-	r         retry.Retrier
-	it        id.SortedTagIterator
-	tagOpts   models.TagOptions
-	p         pool.ObjectPool
-	m         ingestMetrics
-	logger    *zap.Logger
-	sampler   *sampler.Sampler
-	attemptFn retry.Fn
-	ingestFn  func()
+	s                storage.Appender
+	r                retry.Retrier
+	it               id.SortedTagIterator
+	tagOpts          models.TagOptions
+	p                pool.ObjectPool
+	m                ingestMetrics
+	logger           *zap.Logger
+	sampler          *sampler.Sampler
+	attemptFn        retry.Fn
+	ingestFn         func()
+	storeMetricsType bool
 
 	c           context.Context
```

Comment:
Nit: I know this is prior code, but maybe you could take an opportunity and change all those single character field names here to something comprehensible?